### ISSUE TITLE
Update for compatibility with python-bitcoinrpc 0.3

### ISSUE THREAD

**Issue by LitecoinNode** [litecoin-association/LitecoinNode#20]
python-bitcoinrpc 0.3 no longer uses username and password authentication like the following example

get_lcd_info = AuthServiceProxy("http://'"$RPC_USER"':'"$RPC_PASSWORD"'@127.0.0.1:9332")

We have to update the code to be python-bitcoinrpc 0.3 compatible if we want to update to that version op python-bitcoinrpc

more info for 0.3 here
https://pypi.python.org/pypi/python-bitcoinrpc

**Comment by losh11** [comment]
Isn't auth still like:
```
rpc_connection = AuthServiceProxy("http://%s:%s@127.0.0.1:8332"%(rpc_user, pc_password))
```


Let me check - the only thing this configures is the HTML page right?

**Comment by losh11** [comment]
On [```sed -i -e '10iget_lcd_info = AuthServiceProxy("http://'"$RPC_USER"':'"$RPC_PASSWORD"'@127.0.0.1:9332")\' $HOME/scripts/litecoin-node-status.py```](https://github.com/litecoin-association/LitecoinNode/blob/master/ubuntu/ubuntu-install-statuspage.sh#L49)

Why is it defined as ```10iget_led_info``` instead of ```get_led_info``` as used to extract data onto the webpage?

**Comment by LitecoinNode** [comment]
You are correct, the litecoin-node-status.py generates the html file for node status. To get some data such as connected clients, current connected clients etc. litecoin-node-status.py connects to the rpc interface of Litecoind. To do this litecoin-node-status.py needs to authenticate with a username and password.

In the line you mentioned the "10i" instructs sed to insert the data for the authentication at line 10 of litecoin-node-status.py file

If you edit this you have to edit several files mainly the install and probably the update scripts for debian and ubuntu. Im currently on mobile so im not 100% sure on the exact filenames that need editing.
